frontend/src/components/Agreements/AgreementMetaAccordion/AgreementMetaAccordion.jsx

```diff
@@ -85,10 +92,15 @@ const AgreementMetaAccordion = ({
                         )}
                     </dl>
                     <dl>
-                        {renderTerm(
-                            "procurement-shop",
-                            "Procurement Shop",
-                            `${agreement?.procurement_shop?.abbr} - Fee Rate: ${agreement?.procurement_shop?.fee_percentage ? agreement?.procurement_shop?.fee_percentage : 0}%`
+                        {newAwardingEntity ? (
+                            <div
+                                className="padding-left-1"
+                                style={{ borderLeft: "3px solid #0050D8" }}
```

Comment:
may consider adding this color to our global styles (styles.scss) if its not already there